aws-k8s-cni.yaml

```diff
@@ -0,0 +1,90 @@
+package:
+  name: aws-k8s-cni
+  version: 1.18.3
+  epoch: 0
+  description: Networking plugin repository for pod networking in Kubernetes using Elastic Network Interfaces on AWS
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - busybox
+      - ca-certificates-bundle
+      - go
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/aws/amazon-vpc-cni-k8s
+      tag: v${{package.version}}
+      expected-commit: 128a08e41001e81ed317410bd386954022b941af
+
+  - name: Build aws-cni
```

Comment:
All of these should likely be split into subpackages as `cni-init` also contains components within this repository